tasks/enable_psmdb_repo.yml

```diff
@@ -0,0 +1,20 @@
+  - name: include playbook for removing Percona repo
+    include: remove_percona_repository.yml
+
+  - name: Install percona repository package for deb based distros
+    apt:
+      deb: https://repo.percona.com/apt/percona-release_latest.generic_all.deb
+    when: ansible_os_family == "Debian"
+
+  - name: Install percona repository package for rpm based distros
+    yum: name=https://repo.percona.com/yum/percona-release-latest.noarch.rpm
+    when: ansible_os_family == "RedHat"
+
+  - name: enable {{ psmdb_repo }} repo
+    command: percona-release enable-only {{ psmdb_repo }}
```

Comment:
just be aware that enable-only will disable all other repos, so for example if you want to install PBM or PT (which has 2 mongodb tools in the package) or something else from the tools repo it will fail

Reply:
I planned to use this task for psmdb only, that's why I used "enable-only" directive on purpose to ensure that the correct repo is enabled and all others are disabled (other repos might be enabled due to any unexpected reasons like failed cleanup or smth. else).